lib/pbench/server/s3backup/__init__.py

```diff
@@ -306,7 +306,7 @@ def calculate_multipart_etag(tb, chunk_size):
         md5s = []
 
         with open(tb, "rb") as fp:
-            for data in fp.read(chunk_size):
+            for data in iter(lambda: fp.read(chunk_size), b""):
```

Comment:
Seems a simple `while` loop might have been easier to read here? But, apparently this idiom is official, so, good enough.